packages/directlinespeech/__tests__/sendSpeechActivity.js

```diff
@@ -18,8 +18,8 @@ beforeEach(() => {
   global.AudioContext = MockAudioContext;
 });
 
-test('should echo back when saying "hello" and "world"', async () => {
-  const { directLine, sendTextAsSpeech } = await createTestHarness();
+test.each([true, false])('should echo back when saying "hello" and "world"', async (enableInternalHttpSupport) => {
```

Comment:
We should split these into separate tests and add with/without enableInternalHttpSupport to the description so if one fails its more clear where the issue is.